The same `SC2016` messages pop up. Is it got disabled properly, or is such an `info` considered fatal?

```
In /nix/store/9acd5pn68s73d21vkjn8kl5h3ykavzsv-xenBootBuilder/bin/xenBootBuilder line 30:
mapfile -t preGenerations < <(find "$efiMountPoint"/loader/entries -type f -name 'xen-*.conf' | sort -V | sed 's_/loader/entries/nixos_/loader/entries/xen_g;s_^.*/xen_xen_g;s_.conf$__g')
                                                                                                              ^-- SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.


In /nix/store/9acd5pn68s73d21vkjn8kl5h3ykavzsv-xenBootBuilder/bin/xenBootBuilder line 73:
        xenGen=$(echo "$gen" | sed 's_/loader/entries/nixos_/loader/entries/xen_g;s_^.*/xen_xen_g;s_.conf$__g')
                                   ^-- SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.


In /nix/store/9acd5pn68s73d21vkjn8kl5h3ykavzsv-xenBootBuilder/bin/xenBootBuilder line 136:
mapfile -t postGenerations < <(find "$efiMountPoint"/loader/entries -type f -name 'xen-*.conf' | sort -V | sed 's_/loader/entries/nixos_/loader/entries/xen_g;s_^.*/xen_xen_g;s_.conf$__g')
                                                                                                               ^-- SC2016 (info): Expressions don't expand in single quotes, use double quotes for that.

For more information:
  https://www.shellcheck.net/wiki/SC2016 -- Expressions don't expand in singl...
error: builder for '/nix/store/q7p22gdmdkqpnqymm15kgwjdk4id1lqk-xenBootBuilder.drv' failed with exit code 1
error: 1 dependencies of derivation '/nix/store/08va25dxqyq69vkwyk6hnmyhl0ghgcxk-install-systemd-boot.sh.drv' failed to build
error: 1 dependencies of derivation '/nix/store/2zkhvwaf26iqmsijrmpwjbiyi6pax79r-nixos-system-xen-dom0-24.11pre-git.drv' failed to build
error: 1 dependencies of derivation '/nix/store/q7bksgnlw2bqbcjz25w937zqn05s87b9-morph.drv' failed to build
Error while running `/run/current-system/sw/bin/nix-build /nix/store/pjbhydgi8mz4mk190i0r309r3ga1ihiz-morph-1.7.0-lib/eval-machines.nix --arg networkExpr /root/nixos-deployments/nixos-build-xen-test/morph.nix --argstr argsFile /tmp/morph-4126753695/morph-args.json --out-link /tmp/morph-4126753695/result --attr machines ...`: See above.
```

--- a/nixos/modules/virtualisation/xen-dom0.nix
+++ b/nixos/modules/virtualisation/xen-dom0.nix
@@ -32,6 +32,10 @@ let
     runtimeEnv = {
       efiMountPoint = config.boot.loader.efi.efiSysMountPoint;
     };
+
+    # We disable SC2016 because we don't want to expand the regexes in the sed commands.
+    excludeShellChecks = [ "SC2016" ];
+
     text = builtins.readFile ./xen-boot-builder.sh;
   };
 in

